docs/source/reference/finetuning/rl_with_openpipe.md

```
@@ -0,0 +1,603 @@
+      delete_old_checkpoints: false
+      init_args:
+        max_seq_length: 8192
+      engine_args:
+        gpu_memory_utilization: 0.9
+        tensor_parallel_size: 1
+    training:
+      learning_rate: 1e-6
+      beta: 0.0
+```
+
+**Backend Configuration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)